layouts/partials/parts-tools.de.md

```diff
@@ -5,11 +5,16 @@ spezialisierteste Werkzeug ist die Crimpzange, welche dein lokaler Hackspace /
 Maker Space / Fab Lab sicherlich rumliegen hat. Eventuell musst du also nicht
 jedes dieser Werkzeuge kaufen, wenn du eines ausleihen kannst.
 
-* [Crimpzange](https://www.amazon.de/gp/product/B07VX6YGQ8) für JST-XH Steckverbinder (andere Marken oder Zangentypen, die mit diesen Verbindern kompatibel sind, funktionieren auch, z.B. Engineer PA-09).
-* Ein guter, regelbarer Lötkolben oder eine Lötstation, mit entsprechender Lösung zur Entlüftung
-* Pinzetten, Seitenschneider, Abisolierzange, M3 Sechskantschraubendreher ("Inbussschlüssel")
+* [Crimpzange](https://www.amazon.de/gp/product/B07VX6YGQ8) für JST-XH 
+  Steckverbinder (andere Marken oder Zangentypen, die mit diesen Verbindern 
+  kompatibel sind, funktionieren auch, z.B. Engineer PA-09).
+* Ein guter, regelbarer Lötkolben oder eine Lötstation, mit entsprechender 
+  Lösung zur Entlüftung
+* Pinzetten, Seitenschneider, Abisolierzange, M3 Sechskantschraubendreher 
+  ("Inbussschlüssel")
```

Comment:
Inbusschlüssel isn't Flussschifffahrt, there is an extra s in there ;)

Reply:
For this, I only added the .de so that I could create a .en version. I can remove the extra s though. That's a good catch.